client/client.cpp

```diff
@@ -123,7 +124,9 @@ void ClientImpl::registrationFinished(QDBusPendingCallWatcher *watcher)
 
 Action *ClientImpl::addClientAction(const QString &shortcut, const QString &path, const QString &description, QObject *parent)
 {
-    if (!QRegExp(QStringLiteral("(/[A-Za-z0-9_]+){2,}")).exactMatch(path))
+    // FIXME: check if QRegExp syntax is the same
+    static QRegularExpression regexp(QStringLiteral("(/[A-Za-z0-9_]+){2,}"));
+    if (!regexp.match(path).hasMatch())
```

Comment:
@gfgit 
`QRegularExpression::anchoredPattern` should be used for exact matching. After using it, please remove the "FIXME" comment.

Reply:
How about merge it and fix it afterwards?
It's the last piece missing for having all qt6-ports in git and we need more testers IMO.

Reply:
I can fix it today

Reply:
Done. What about duplicated `joinCommandLine()` method?